app/routes/how-can-i-help.knowledge.tsx

```diff
@@ -28,6 +30,18 @@ const NewToAISafety = () => (
           {
             title: 'Intro to AI safety video playlist',
             action: 'https://aisafety.info/questions/9OGZ/',
+            image: (
+              <div style={{position: 'relative'}}>
```

Comment:
this is very ugly and not the Right Way to do this kind of thing, but if it's a one off kind of thing it should be fine?